index.html

```diff
@@ -34,13 +36,14 @@
         }
 
         #theme-toggle {
-            margin-top: 5px;
-            padding: 10px;
+            margin-top: 15px;
```

Comment:
Could we consider to use rem instead of px?

Reply:
done